blackbox/docs/appendices/release-notes/3.2.5.rst

```diff
@@ -0,0 +1,61 @@
+.. _version_3.2.5:
+
+=============
+Version 3.2.5
+=============
+
+Released on 2019/03/11.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 2.0.4 or higher
+    before you upgrade to 3.2.5.
+
+    We recommend that you upgrade to the latest 3.1 release before moving to
+    3.2.5.
+
+    If you want to perform a `rolling upgrade`_, your current CrateDB version
+    number must be at least :ref:`version_3.2.0`. Any upgrade from a version
+    prior to this will require a `full restart upgrade`_.
+
+    When restarting, CrateDB will migrate indexes to a newer format. Depending
+    on the amount of data, this may delay node start-up time.
+
+    Please consult the :ref:`version_3.0.0_upgrade_notes` before upgrading.
+
+.. WARNING::
+
+    Tables that were created prior to upgrading to CrateDB 2.x will not
+    function with 3.2 and must be recreated before moving to 3.2.x.
+
+    You can recreate tables using ``COPY TO`` and ``COPY FROM`` while running a
+    2.x release into a new table, or by `inserting the data into a new table`_.
+
+    Before upgrading, you should `back up your data`_.
+
+.. _rolling upgrade: http://crate.io/docs/crate/guide/best_practices/rolling_upgrade.html
+.. _full restart upgrade: http://crate.io/docs/crate/guide/best_practices/full_restart_upgrade.html
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+.. _inserting the data into a new table: https://crate.io/docs/crate/reference/en/latest/admin/system-information.html#tables-need-to-be-recreated
+
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Changelog
+=========
+
+Fixes
+-----
+
+- Fixed an issue that could cause queries on the :ref:`sys.snapshots
+  <sys-snapshots>` table to raise an error if a snapshot couldn't be retrieved.
+
+- Fixed an issue that caused inserts into partitioned tables to fail with an
+  ``unknown setting`` error if the table was created in an earlier version of
+  CrateDB using settings that have been removed in later versions.
+
+- Fixed an issue that could result in a ``ConcurrentModificationException``
+  error when querying the ``sys.jobs_metrics`` table.
```

Comment:
"while" instead of "when"

Reply:
I'll make these changes in a seperate PR against master. Little point changing them in this PR.

Reply:
ok. marked as resolved here. added some additional comments. not sure if you want to do them on a second PR also